src/Umbraco.Web.UI.Client/src/less/application/grid.less

```diff
@@ -217,19 +223,17 @@ body {
     padding-left:20px
 }
 
-
-
 @media (max-width: 767px) {
 
     // make leftcolumn smaller on tablets
     #leftcolumn {
-      width: 60px;
+        width: 61px;
     }
     #applications ul.sections {
-        width: 60px;
+        width: 61px;
     }
     ul.sections.sections-tray {
-        width: 60px;
+        width: 61px;
```

Comment:
I am not quite sure why you add 1px to the width of sections. Then the total will be 62, 42 etc with the border and then the margin-left on the overlay will be 1px off again?